Apps/phdigicert/readme.html

```diff
@@ -0,0 +1,3 @@
+<html><head></head><body>
```

Comment:
Add license header above

Reply:
@pdesai-crest Done with the said changes